REQUIRE

```diff
@@ -1,4 +1,4 @@
 julia 0.3
 Memoize
 Compat
```

Comment:
Add the minimum `Compat` version requirement that introduced the features you're using.